warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -120,7 +123,8 @@ public static void main(String[] args) throws AccumuloSecurityException, IOExcep
             log.error("usage: BulkIngestMapFileLoader hdfsWorkDir jobDirPattern instanceName zooKeepers username password "
                             + "[-sleepTime sleepTime] [-majcThreshold threshold] [-majcCheckInterval count] [-majcDelay majcDelay] "
                             + " [-seqFileHdfs seqFileSystemUri] [-srcHdfs srcFileSystemURI] [-destHdfs destFileSystemURI] [-jt jobTracker] "
-                            + "[-ingestMetricsDisabled] [-jobObservers jobObserverClasses] [-shutdownPort portNum] confFile [{confFile}]");
+                            + "[-ingestMetricsDisabled] [-jobObservers jobObserverClasses] [-shutdownPort portNum] [-delayPathPattern pattern] "
```

Comment:
-delayPathPattern or -delayPattern ?  change usage or if statement below.

Reply:
So delay path pattern is a glob against the files under the bulk load directory if I understand the code correctly.  This should be made clear in the comments.  At least add a log.info line to the output around line 340 below.